### ISSUE TITLE
[Request] Add a direction prop to set which way the picker opens

### ISSUE THREAD

**Issue by MrLeebo** [YouCanBookMe/react-datetime#171]
This is a new feature request. If you position the Datetime control towards the bottom of the page, the picker will appear below it and off the page entirely, forcing the user to scroll before they can make a selection. I hope it would not be too much trouble to add a `direction` prop which could take `oneOf(['up', 'down', 'left', 'right'])` and position the picker in that direction relative to the input.

```
<Datetime direction="up" />
```

**Comment by simeg** [comment]
Hi @MrLeebo. I've also had the specific problem you are mentioning, you can solve it using JS. Maybe even HTML using an anchor? But about your suggestion, I think the idea is good. If you'd like to create the code I'd be happy to review the PR.

**Comment by MrLeebo** [comment]
I created the issue because I don't think I have time to learn the repo and implement this right now, but if that changes I will open up a PR.

**Comment by danosborne** [comment]
+1

Will also try to spin up a PR...

**Comment by simeg** [comment]
A tip: use react-date-playground for development (check the README).

**Comment by danosborne** [comment]
Hacked for 5 mins and opened the PR above. Are people happy with the approach? 

If so I will tidy it up and actually follow the contributing guidelines etc. ;-)

**Comment by Krzyku** [comment]
For me `direction` prop is misleading. I understand direction as left-to-right or right-to-left settings.
Maybe `position` is better idea for name of this feature?

**Comment by bjbrewster** [comment]
I agree. Bootstrap Popover/Tooltip/Overlay uses prop `placement` with values `top`, `bottom`, `left` and `right`. Can we use the same prop name with `top` and `bottom`?

**Comment by simeg** [comment]
`placement` sounds good to me. Will try to find time for looking into this soon.

**Comment by kostiantyn-solianyk** [comment]
any updates?

**Comment by Bramgus12** [comment]
Is there a workaround to get the date picker above the input field?

**Comment by jvnlwn** [comment]
To move the date picker above the input, just add `bottom: 100%;` style to the `. rdtPicker` class. (Caveat in case it matters: this is version 2.14.0 pictured.)

<img width="269" alt="Screen Shot 2022-09-06 at 11 19 39 AM" src="https://user-images.githubusercontent.com/5227102/188673179-af6092c7-6eff-450e-ab37-018d83ef920f.png">
